### PR TITLE
Page sécurité fr

### PR DESCRIPTION
**Pull request by adeclosset**
Uploading new security French page

### CODE REVIEW

**Comment by adeclosset** [comment]
@steve - want to merge?

File: app/pages/fr/pro/index.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h2 class="u-text-heading u-color-heading u-text-l u-text-light">
             Découvrez GoCardless Pro
           </h2>
           <p class="u-text-s u-color-p u-margin-Txs">
-            Conçu pour les grandes entreprises, GoCardless Pro combine simplicité d'utilisation et contrôle complet du paiement et de l'expérience client. Vous pouvez encaisser vos paiement dans toute la zone SEPA et le Royaume-Uni en une seule intégration.
+            Conçu pour les grandes entreprises et start-ups en forte croissance, GoCardless Pro combine simplicité d'utilisation et contrôle complet du paiement et de l'expérience client. Vous pouvez encaisser vos paiement dans toute la zone SEPA et le Royaume-Uni en une seule intégration.
```
Comment (stevedomin) [inline app/pages/fr/pro/index.html]: **à** forte croissance?

**Comment by adeclosset** [comment]
@stevedomin Ok done, sorry did not know where the comments were (still learning Github!)

**Comment by stevedomin** [comment]
No worries.

**Pull request merged**